include/verilatedos.h

```diff
@@ -475,6 +475,8 @@ using ssize_t = uint32_t;  ///< signed size_t; returned from read()
 #  define VL_CPU_RELAX() asm volatile("yield" ::: "memory")
 # elif defined(__powerpc64__)
 #  define VL_CPU_RELAX() asm volatile("or 1, 1, 1; or 2, 2, 2;" ::: "memory")
+# elif defined(__loongarch__)
+#  define VL_CPU_RELAX() asm volatile("" ::: "memory")
```

Comment:
Please make this whatever instruction does yield on this architecture. There's sure to be something.  Without that there may be performance loss and power waste.

Reply:
Because LoongArch doesn't have a instruction similar to "yield". Hope you can accept my use of the "nop" instruction.